examples/usecases/ranking_with_multitask_learning.ipynb

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
<p>"projecting the inputs independently and then having the experts output<s>s</s> weighted averaged by a gate <s>to</s> for a shared representation to be used for all tasks" (two typos, s and to are unnecessary)</p><p> </p><p>"gate for each task, so that they could choose how <strong>to </strong>better combine the experts outputs" ("to" missing)</p><p> </p><p>" dataset<s> by</s> with just a few lines of code!" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/948/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>fixed</p>